gdx/src/com/badlogic/gdx/graphics/ImmutableColor.java

```diff
@@ -0,0 +1,124 @@
+
+package com.badlogic.gdx.graphics;
+
+/** A color class that prevents mutation to r/g/b/a values. */
+public class ImmutableColor extends Color {
```

Comment:
ImmutableColor should probably be `final` to ensure there isn't a mutable subclass created.